installation/on_debian_and_ubuntu.md

```diff
@@ -33,6 +33,7 @@ sudo apt install libxml2-dev     # for using XML
 sudo apt install libyaml-dev     # for using YAML
 sudo apt install libgmp-dev      # for using Big numbers
 sudo apt install libreadline-dev # for using Readline
+sudo apt install libz-dev        # for using crystal play
```

Comment:
```suggestion
sudo apt install libz-dev        # for using Crystal play
```

Reply:
I think this refers to the command (`crystal play`), otherwise it would be "the Crystal playground".

Reply:
right